platform/windows/display_server_windows.cpp

```diff
@@ -914,6 +914,31 @@ String DisplayServerWindows::clipboard_get() const {
 				GlobalUnlock(mem);
 			}
 		}
+		
+	} else if (IsClipboardFormatAvailable(CF_HDROP)) {
+		HGLOBAL mem = GetClipboardData(CF_HDROP);
+		if (mem != nullptr) {
+			HDROP hDropInfo = (HDROP)GlobalLock(mem);
+			if (hDropInfo != nullptr) {
+				const int buffsize = 4096;
+				WCHAR buf[buffsize];
+
+				int fcount = DragQueryFileW(hDropInfo, 0xFFFFFFFF, nullptr, 0);
+
+				Vector<String> files;
+
+				for (int i = 0; i < fcount; i++) {
+					DragQueryFileW(hDropInfo, i, buf, buffsize);
+					// since we return a string, there's no easy way to understand what is intended to be just a string, or a file path.
+					// so we use the 'file://' URI to make it clear.
+					// technically two slashes is invalid yet supported, and linux returns two, so we use two for portability.
+					String file = "file://" + String::utf16((const char16_t *)buf);
```

Comment:
Simply adding `file://` will result in a path with mixed separators, Windows APIs return Windows style path with drive letter and `\` as separators or `//?/` UNC path, so it should be either converted to proper `file` URL scheme (preferred behavior, e.g., `file:///c:/windows/filename`) or left as is.

Reply:
forcing it to three slashes may break accessing files on remote drive paths, but i don't know the pattern for how windows returns a remote file path so i just thought i'd choose what probably wouldn't break

I wasn't sure about which separator style to go with, since trying to do anything with the file with just `file://` worked for my cases
im sorry if these seem like im being sloppy

Reply:
UNC paths work with `file://` as far as i can tell, I only have a local path for WSL to check against but it doesn't appear to fail. This shouldn't be a problem because the code is only for files on the clipboard which in windows should either be regular paths or UNC.